xinference/model/llm/llama_cpp/core.py

```diff
@@ -131,6 +132,10 @@ def load(self):
                 legacy_model_file_path = os.path.join(self.model_path, "model.bin")
                 if os.path.exists(legacy_model_file_path):
                     model_path = legacy_model_file_path
+            if self.model_spec.multimodal_projectors:
+                mmproj = os.path.join(
+                    self.model_path, self.model_spec.multimodal_projectors[0]
```

Comment:
Maybe we can let the users pass `multimodel_projector` to specify the exact one?